common/djangoapps/course_category/models.py

```diff
@@ -0,0 +1,74 @@
+from django.db import models
+from django.db.models.signals import m2m_changed, pre_delete, post_delete
+from django.dispatch import Signal, receiver
+from django.utils.translation import ugettext_lazy as _
+
+from mptt.models import MPTTModel, TreeForeignKey
+from openedx.core.djangoapps.xmodule_django.models import CourseKeyField
+
+from .tasks import task_reindex_courses
+
+
+move_to = Signal()
```

Comment:
I would name Signal instance as `signal` to make it be more clear in the code.

Reply:
Done